runner/__init__.py

```diff
@@ -192,6 +203,9 @@ def make_argument_parser():
     parser.add_argument("-n", "--times", dest="times", type=int, help="run this many times (default is forever)")
     parser.add_argument("-H", "--halt-after", dest="halt_after", action="store_const", const=True,
                         help="Call the halt task after runner finishes (never called if -n is not set).")
+    parser.add_argument("--syslog", dest="syslog", action="store_const", const=True, help="send messages to syslog")
+    parser.add_argument("--syslog-address", dest="syslog_address", default=get_syslog_address(),
```

Comment:
https://docs.python.org/2/library/logging.handlers.html#logging.handlers.SysLogHandler

I'd prefer to keep it called address, since that's what they chose to call it in the docs. It's always awkard to deal with a function that behaves differently depending on the argument type.

That in mind, I think it's best to just now add the ability to configure the address for now. If we need to log to a syslog server later, we can add the arguments. Otherwise it's just cruft imo.